.github/workflows/create_sbom_report.yml

```diff
@@ -0,0 +1,119 @@
+            if ((Get-Content ${{ matrix.path }}imagedata.json | jq '.[1] | .detail') -match '\d{8}\.+\d') { 
+              if ($Matches[0] -eq ${{ github.event.client_payload.imageLabel }}) {
+                echo "Current runner use ${{ github.event.client_payload.imageLabel }} image version."
+              } else {
+                echo "Error. Current runner image version don't match ${{ github.event.client_payload.imageLabel }} image version."
+                exit 1
+              }
+            } else {
+              echo "Cannot parse image version."
+              exit 1
+            }
+          }
+  #Install and run SYFT, compress SBOM, upload it to release assets
+  create-sbom:
+    strategy:
+      matrix:
+        include:
+          - os: Windows
+            path: D:/syft
+          - os: Linux
+            path: /usr/local/bin
+          - os: macOS
+            path: /usr/local/bin
+    defaults:
+      run:
+        shell: pwsh
+    needs: [initialize, version-check]
+    runs-on: ${{ initialize.output.current-os }}
+    steps:
+      - name: Install SYFT tool
+        run: |
+          echo "Trying to install SYFT tool from oficial sources"
+          if ("${{ runner.os }}" -eq "${{ matrix.os }}") {
+            echo "Installing SYFT for ${{ matrix.os }}"
+            curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b ${{ matrix.path }}
+          }
+      - name: Prepare SBOM report
+        run: |
+          if ("${{ runner.os }}" -eq "Windows") {
+            echo "Runnig SYFT tool on Windows on disk C:/"
+            D:/syft/syft dir:C:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json
+          } elseif ("${{ runner.os }}" -eq "macOS") {
+              echo "Runnig SYFT tool on macOS with exludes"
+              syft dir:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json --exclude ./Users --exclude ./System/Volumes --exclude ./private
+          } else {
+                echo "Runnig SYFT tool on Ubuntu with default settings"
+                syft dir:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json
+          }
+          Compress-Archive sbom.${{ github.event.client_payload.imageLabel }}.json sbom.${{ github.event.client_payload.imageLabel }}.json.zip
+          echo "SBOM created successfully!"
+      #Might be changed to softprops/action-gh-release after additional check
+      - name: Upload release asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: "https://uploads.github.com/repos/actions/runner-images/releases/${{ github.event.client_payload.GHreleaseID }}/assets{?name,label}"
+          asset_path: ./SBOM.${{ github.event.client_payload.imageLabel }}.json.zip
+          asset_name: SBOM.${{ github.event.client_payload.imageLabel }}.json.zip
```

Comment:
Just for curiosity - why do we need to include imageLabel to file name?

Reply:
Discussed offline.

Reply:
After further discussion, we came to the following conclusion:
It is desirable to have unique report names for current versions of operating systems. Assumed applicability scenario: the user wants to compare the reports of two different operating systems, the downloaded files will have a visual difference, helping to determine "where to look."
On reflection, there is one interesting situation where it would be necessary to add another token with the current image version to the name.